WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1790,13 +1780,20 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     break;
                 case RequestCodes.VIDEO_LIBRARY:
                     Uri videoUri = data.getData();
-                    fetchMedia(Arrays.asList(videoUri));
```

Comment:
This isn't an issue introduced in this PR, but shouldn't we be checking `resultCode == Activity.RESULT_OK` here?